buildenv/jenkins/tapVerification/TapCollection.groovy

```diff
@@ -0,0 +1,88 @@
+pipeline {
+    agent { label 'ci.role.test&&hw.arch.x86&&sw.os.linux' }
+    parameters {
+        string(name: 'RELEASE_TRIAGE_ISSUE_NUMBER', defaultValue: '', description: 'Triage issue number')
+        string(name: 'RELEASE_TRIAGE_REPO', defaultValue: 'adoptium/aqa-tests', description: 'Triage repo')
```

Comment:
Do you mind to put the TRIAGE_REPO first, and the TRIAGE_ISSUE_NUMBER after it?  Matches the ordering we use in other pipelines and in the case of jobName followed by jobNumber

Reply:
Updated